sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -397,9 +408,16 @@ private[ui] class ExecutionPagedTable(
 
     val desc = if (execution.description != null && execution.description.nonEmpty) {
       <a href={executionURL(execution.executionId)} class="description-input">
-        {execution.description}</a>
+        <span
+        data-toggle="tooltip" data-placement="top" title="Click to navigate query details page.">
+          {execution.description}
+        </span></a>
```

Comment:
We already added the tooltip in headers. Right? Is it necessary to add for every row, the same information?

Reply:
I think yes, because there link is getting displayed, which is accessible.

Reply:
I would not add tool tips to every row. It will be distracting to have these constantly popping up all over the table.

Reply:
@07ARB Yes, I think both cases we need not show the tool tips for every row as we are already showing in the header.

Reply:
Ok